XenModel/Network/NetworkingHelper.cs

```diff
@@ -94,7 +94,7 @@ public static List<PIF> GetAllPhysicalPIFs(Host host)
         /// The PIFs will be sorted by name, and deduplicated by name -- i.e. there will be only
         /// one per device name, even if there are multiple hosts in the pool.
         /// 
-        /// Note that this is not the same as GetAllPhysicalPIFs(pool.master) because the master
+        /// Note that this is not the same as GetAllPhysicalPIFs(pool.master) because the coordinator

```

Comment:
master

Reply:
Left in this case because while this is a comment, it's referring to a call to the XenAPI project.